config/initializers/form_errors.rb

```diff
@@ -0,0 +1,11 @@
+# Add error span to form fields with errors
+Rails.application.config.action_view.field_error_proc = proc do |html_tag, instance|
```

Comment:
Does `ActionView::Base.field_error_proc` work here?

Reply:
Looks like it does! Just swapped it out